cypress.config.ts

```diff
@@ -21,6 +23,12 @@ export default eyesPlugin(
         // copy any needed variables from process.env to config.env
         config.env.useAppli = process.env.USE_APPLI ? true : false;
 
+        // Argos
+        registerArgosTask(on, config, {
+          uploadToArgos: !!process.env.CI,
+          token: 'fc3a35cf5200db928d65b2047861582d9444032b',
```

Comment:
Should not it be in the environment as well?

Reply:
Of course but I can't do it for you!

Reply:
I can set it in the env. Please add me to the team in Argos so I can regenerate it.

Reply:
It's not possible to regenerate it yet. 😅

Reply:
We can clean the history with force push

Reply:
Yeah it's not very sensible 😉